cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/web/ResultsController.java

```diff
@@ -26,28 +25,18 @@
 
     @RequestMapping(value = "/search")
     public String search(@RequestParam("search") String query, Model model,
-            @RequestParam("page") Optional<Integer> currentPage, @RequestParam("size") Optional<Integer> size,
-            @RequestParam("totalPages") Optional<Long> totalPageNumbers) {
+            @PageableDefault(size = 20) Pageable pageable) {
         int pageSize = Integer.parseInt(env.getRequiredProperty("page.size"));
```

Comment:
The annotation accepts a constant final value, not sure how to make it configurable with the pageSize property.